lib/auth/grpcserver.go

```diff
@@ -2582,7 +2582,7 @@ func userSingleUseCertsGenerate(ctx context.Context, actx *grpcContext, req prot
 		ctx, req,
 		certRequestMFAVerified(mfaDev.Id),
 		certRequestPreviousIdentityExpires(actx.Identity.GetIdentity().Expires),
-		certRequestClientIP(clientIP),
+		certRequestLoginIP(clientIP),
```

Comment:
This behaviour here is incorrect, the single use certs should be pinned to the client's IP. However, this is a long-standing bug, so there's no issue with your change here - just wanted to make sure we don't lose track of this and come back to ensure that we correct the behaviour of the GenerateUserCerts RPC :)

Reply:
Yep, I remember it 👍  :) 
Should be working as originally intended in the end.